pkg/internal/checkup/checkup.go

```diff
@@ -47,3 +75,41 @@ func (c *Checkup) Teardown(ctx context.Context) error {
 func (c *Checkup) Results() status.Results {
 	return status.Results{}
 }
+
+func newRealtimeVMI(checkupConfig config.Config) *kvcorev1.VirtualMachineInstance {
+	const (
+		rootDiskName      = "rootdisk"
+		cloudInitDiskName = "cloudinitdisk"
+		userData          = `#cloud-config
+	password: redhat
+	chpasswd:
+	  expire: false
+	user: user`
+	)
+
+	newVMI := vmi.NewBuilder(randomizeName(VMINamePrefix)).
```

Comment:
This is another builder pattern.
I do not understand why you had to re-invent another one when we already have one in use in the reference checkup.

I would expect to re-use that one instead. And if there is a good reason, to invent a new one.

I personally do not like this "train" pattern, it reminds Java builders.
While the K8S clientset uses this pattern, it uses it in a pretty fixed format (i.e. the order makes sense and is not endless).

Reply:
Changed the code to use a function, similar to the VM latency checkup.
Removed the builder object.